api/src/routes/cas.ts

```diff
@@ -0,0 +1,55 @@
+/** Node Imports */
+import express from 'express';
+import bodyParser from 'body-parser';
+import CASAuthentication from 'express-cas-authentication';
+
+/** Custom Imports */
+import {
+  CasAPIController,
+  CasUserController,
+  CasAuthController,
+  CasLogoutController,
+} from '../controllers/cas';
+
+/** Instantiate CASAuthentication */
+const cas = new CASAuthentication({
+  /* eslint-disable-next-line @typescript-eslint/camelcase */
+  cas_url: 'https://cas-auth.rpi.edu/cas',
+  /* eslint-disable-next-line @typescript-eslint/camelcase */
+  service_url: process.env.NODE_ENV === 'production' ? 'https://rpicampusmap.herokuapp.com' : 'http://localhost:3000',
+  /* eslint-disable-next-line @typescript-eslint/camelcase */
+  session_name: 'casUser',
```

Comment:
You can do `/* eslint-disable @typescript-eslint/camelcase */` one line 16 and then `/* eslint-enable @typescript-eslint/camelcase */` after the `session_name` property to make this cleaner